js/adapt-assessmentArticleView.js

```diff
@@ -29,6 +30,26 @@ define([
             this.stopListening(Adapt, "assessments:reset", this._onAssessmentReset);
         },
 
+        setupQuestionNumbering: function() {
+            Handlebars.registerHelper("questionNumber", function getQuestionNumber() {
+                var currentArticle = this._parent.attributes._parentId;
+                var currentAssessment =  Adapt.assessment.get().filter(function(assessment) {
+                    return assessment.attributes._id === currentArticle;
+                })[0];
+                var questionComponents = currentAssessment._currentQuestionComponents;
+                return questionComponents.indexOf(Adapt.findById(this._id)) + 1;
```

Comment:
Could use the `model` from earlier instead of another lookup with `Adapt.findById()`.